go/prometheusexporter/collector/collector.go

```diff
@@ -477,6 +490,38 @@ func (c *Collector) registerExceptionsTotalVec() {
 	c.exceptionsRegistry.MustRegister(c.actionMetrics.exceptionsTotalVec)
 }
 
+func (c *Collector) registerWebVitalsMetrics() {
+	c.actionMetrics.webvitalsCls = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "logjam:action:webvitals_cls_score",
+			Help:    "measured Cumulative Layout Shift",
+			Buckets: []float64{0.01, 0.025, 0.050, 0.1, 0.25, 0.5, 1},
+		},
+		[]string{"app", "env", "action"},
+	)
+	c.registry.MustRegister(c.actionMetrics.webvitalsCls)
+
+	c.actionMetrics.webvitalsFid = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "logjam:action:webvitals_fid_seconds",
+			Help:    "First Input Delay in seconds",
+			Buckets: []float64{0.005, 0.010, 0.025, 0.050, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 25, 50, 100, 250},
+		},
+		[]string{"app", "env", "action"},
+	)
+	c.registry.MustRegister(c.actionMetrics.webvitalsFid)
+
+	c.actionMetrics.webvitalsLcp = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "logjam:action:webvitals_lcp_seconds",
```

Comment:
The other logjam metrics seem to include the `_distribution_` infix for histogram metrics. Should we add this here as well?